Peer.Connectors/GithubPullRequestSource.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Octokit;
+using Peer.Domain;
+using Peer.Domain.Models;
+using Peer.User;
+
+namespace Peer.Connectors
+{
+    public class GithubPullRequestSource : IPullRequestSource<GithubPullRequestSource>
+    {
+        private static readonly Regex _paramRegex = new Regex(@"(?<orgName>[\w\-]+)/(?<repoName>[\w\-]+)/issues/\d+");
+        private readonly AppConfig _config;
+        private static IClient<UserGithubClient> _userClient;
+        private static IGitHubClient _githubClient;
+
+        public GithubPullRequestSource(AppConfig config, IClient<UserGithubClient> userClient)
+        {
+            _config = config;
+            _userClient = userClient;
+        }
+
+        public async Task<IEnumerable<PeerPullRequest>> FetchPullRequestsAsync()
+        {
+            _githubClient = _userClient.CreateClient().GitClient;
+
+            var issues = await _githubClient.Search.SearchIssues(new SearchIssuesRequest()
+            {
+                Archived = false,
+                Involves = _config.Github.Name,
```

Comment:
From the usage here `Name` obviously refers to username (or I've just made myself look like a fool); but under a config section that's just called `Github` would `Username` be more appropriate/specific?